src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,27 +9,266 @@ logo: cloudcannon
 i18nReady: true
 ---
 
-import Grid from '~/components/FluidGrid.astro'
-import Card from '~/components/ShowcaseCard.astro'
+import Grid from '~/components/FluidGrid.astro';
+import Card from '~/components/ShowcaseCard.astro';
+import { Steps, FileTree } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
-[CloudCannon](https://cloudcannon.com) is a Git-based headless content management system that provides a visual editor for your content.
+[CloudCannon](https://cloudcannon.com) is a Git-based headless content management system that provides a visual editor for your content and UI components, providing a rich, live editing experience.
+
+:::tip
+If you're starting a project from scratch, you can use the [CloudCannon Astro Starter Template](https://cloudcannon.com/templates/astro-starter/) with commonly used CloudCannon features to cut down on setup time.
+:::
+
+## Integrating with Astro
+
+This guide will describe the process of configuring CloudCannon as a CMS for Astro using the CloudCannon Site Dashboard.
+
+The Site Dashbaord provides you with an organized view of your Astro files and the ability to edit them using:
+
+- A [Data Editor](https://cloudcannon.com/documentation/articles/the-data-editor/) for managing structured data files and markup.
+- A [Content Editor](https://cloudcannon.com/documentation/articles/the-content-editor/) for WYSIWYG rich text editing in a minimal view.
+- A [Visual Editor](https://cloudcannon.com/documentation/articles/the-visual-editor/) for an interactive preview of your site, allowing you to edit directly on the page.
+
+You can also configure role-based access to a minimal [Source Editor](https://cloudcannon.com/documentation/articles/the-source-editor/), an in-browser code editor for making minor changes to the source code of your files.
+
+## Prerequisites
+
+{/* TODO: Does CloudCannon want a particular UTM link? */}
```

Comment:
Reminder comment to delete once we have our answer!

Reply:
Removing comment now! We can remember to add back in if and when necessary!